README.md

```diff
@@ -12,99 +14,107 @@ by default.
+    - [Docker permissions error](#docker-permissions-error)
+    - [How to run public RPC api](#how-to-run-public-rpc-api)
+    - [RaspberyPI](#raspberypi)
+    - [Run all components by docker-compose](#run-all-components-by-docker-compose)
+      - [Optional: Setup dedicated user](#optional-setup-dedicated-user)
+      - [Environment Variables](#environment-variables)
+      - [Run](#run)
+    - [How to change db pagesize](#how-to-change-db-pagesize)
+    - [Erigon3 perf tricks](#erigon3-perf-tricks)
+    - [Windows](#windows)
 - [Getting in touch](#getting-in-touch)
-    + [Erigon Discord Server](#erigon-discord-server)
-    + [Reporting security issues/concerns](#reporting-security-issues/concerns)
-    + [Team](#team)
+    - [Erigon Discord Server](#erigon-discord-server)
+    - [Blog](#blog)
+    - [Twitter](#twitter)
+    - [Reporting security issues/concerns](#reporting-security-issuesconcerns)
 - [Known issues](#known-issues)
-    + [`htop` shows incorrect memory usage](#htop-shows-incorrect-memory-usage)
+    - [`htop` shows incorrect memory usage](#htop-shows-incorrect-memory-usage)
+    - [Cloud network drives](#cloud-network-drives)
+    - [Filesystem's background features are expensive](#filesystems-background-features-are-expensive)
+    - [Gnome Tracker can kill Erigon](#gnome-tracker-can-kill-erigon)
+    - [the --mount option requires BuildKit error](#the---mount-option-requires-buildkit-error)
 
 <!--te-->
 
-**Disclaimer**: this software is currently a tech preview. We will do our best to keep it stable and make no breaking
-changes but we don't guarantee anything. Things can and will break.
-
-**Important defaults**: Erigon is an Archive Node by default (to remove history see: `--prune` flags
-in `erigon --help`). We don't allow change this flag after first start.
+**Important defaults**: Erigon is an Archive Node by default: use `--prune.mode` if need make it smaller (not allowed to
+change after first start)
 
 <code>In-depth links are marked by the microscope sign (🔬) </code>
 
 System Requirements
 ===================
 
-* For an Archive node of Ethereum Mainnet we recommend >=3.5TB storage space: 2.3TiB state (as of March 2024),
-  643GiB snapshots (can symlink or mount folder `<datadir>/snapshots` to another disk), 200GB temp files (can symlink or
-  mount folder `<datadir>/temp` to another disk).
-  Ethereum Mainnet Full node (see [Pruned Node][pruned_node]): 1.1TiB not including temp files (June 2024).
+RAM: >=32GB, [Golang >= 1.22](https://golang.org/doc/install); GCC 10+ or Clang; On Linux: kernel > v4. 64-bit
+architecture.
 
-* Gnosis Chain Archive: 1.7TiB (March 2024).
-  Gnosis Chain Full node (see [Pruned Node][pruned_node]): 300GiB (June 2024).
-
-* Polygon Mainnet Archive: 8.5TiB (December 2023).
-  Polygon Mainnet Full node (see [Pruned Node][pruned_node]) with `--prune.*.older 15768000`: 5.1Tb (September 2023).
+- ArchiveNode Ethereum Mainnet: 2TB (April 2024). FullNode: 1.1TB (June 2024)
```

Comment:
Maybe some fresh date from our nodes?

Reply:
We don't know - will we release "memoized" domains for bor-mainnet or not. They are 2x bigger.